playstore/playstore.py

```diff
@@ -619,6 +628,39 @@ def download(self, package_name: str, file_name: str = None, download_obb: bool
         if not self._check_entire_file_downloaded(apk_size, file_name):
             return False
 
+        if split_apks:
+            # Save the split apks for the apk.
+            for cfg_apk in split_apks:
+
+                # Execute another query to get the actual file.
+                response = requests.get(cfg_apk.downloadUrl, headers=headers, cookies=cookies, verify=True, stream=True)
+
+                chunk_size = 1024
+                file_size = int(response.headers['content-length'])
+
+                cfg_apk_file_name = os.path.join(os.path.dirname(file_name),
+                                             '{0}.{1}.{2}.apk'.format(package_name, version_code, cfg_apk.name))
```

Comment:
Using the same naming convention as for `obb` files:
```suggestion
                                             '{0}.{1}.{2}.apk'.format(cfg_apk.name, version_code, package_name))
```